content/applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst

```diff
@@ -0,0 +1,73 @@
+===================================
+Configure product for manufacturing
+===================================
```

Comment:
```suggestion
===================================
Manufacturing product configuration
===================================
```
only suggesting this because, in the past, JCS has said we shouldn't start any main doc titles with verbs - but perhaps that's changed - either way, just thought it was worth mentioning